app/app.go

```diff
@@ -459,30 +467,23 @@ func New(
 	app.SetAnteHandler(anteHandler)
 	app.SetEndBlocker(app.EndBlocker)
 
-	planName := "v3.0.0"
-	app.UpgradeKeeper.SetUpgradeHandler(planName, func(ctx sdk.Context, plan upgradetypes.Plan, _ module.VersionMap) (module.VersionMap, error) {
+	planName := "v4.0.0"
+	app.UpgradeKeeper.SetUpgradeHandler(planName, func(ctx sdk.Context, _ upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
 		// a new param in 1.0.0 -- set to roundup(5x 7secs) as a safe choice
 		app.IBCKeeper.ConnectionKeeper.SetParams(ctx, ibcconnectiontypes.NewParams(uint64(40*time.Second)))
-		fromVM := make(map[string]uint64)
-		for moduleName := range app.mm.Modules {
-			fromVM[moduleName] = 1
-		}
-		// delete new modules from the map, for _new_ modules as to not skip InitGenesis
-		delete(fromVM, authz.ModuleName)
-		delete(fromVM, feegrant.ModuleName)
+
```

Comment:
should add interchain accounts?
https://github.com/cosmos/ibc-go/blob/v3.0.0-rc1/docs/migrations/v2-to-v3.md#upgrade-proposal

Reply:
Created an issue for this: https://github.com/crypto-org-chain/chain-main/issues/719
Will do it in future PR.